application/models/ListInsert.php

```diff
@@ -0,0 +1,8 @@
+<?php
+    class ListInsert extends CI_Model{
```

Comment:
in naming Models, name it as a representation of your table in the database. Example: UserModel for users table, BookModel for books table. Or in your case, TodoModel for todo table.